corehq/apps/export/models/new.py

```diff
@@ -2442,12 +2448,17 @@ def _generate_schema_from_case_property_mapping(cls, case_property_mapping, pare
         )
 
         for case_type, case_properties in case_property_mapping.items():
+            if domain_has_privilege(domain, privileges.DATA_DICTIONARY):
+                case_group_name_for_property = get_case_property_group_name_for_properties(domain, case_type)
+            else:
+                case_group_name_for_property = {}
 
             for prop in case_properties:
                 group_schema.items.append(ScalarItem(
                     path=[PathNode(name=prop)],
                     label=prop,
                     last_occurrences={app_id: app_version},
+                    case_property_group_name=case_group_name_for_property.get(prop, None)
```

Comment:
Nit: The default of `None` can be removed since this is already the default return val for `get()`.